### ISSUE TITLE
Extra space (margin) below text in each AsciiDoc table cell

### ISSUE THREAD

**Issue by janicemhdc** [asciidoctor/asciidoctor-pdf#1513]
I will have to come back later with an example if you need one, but I wanted to quickly capture this before I need to log off.

In each of my (admittedly few) PDF documents with tables, each row is sizing to leave blank space at bottom of the row that's the equivalent of an extra row of text. HTML leaves a more reasonable 1/3 of a row or so of space by default.

It seems to me that the default behavior should be similar to the HTML behavior and the current default white space is excessive. That said, is there any way to adjust this without creating a new theme? I'm using the default styling/theme and would prefer to keep it that way.

**Comment by mojavelinux** [comment]
I agree this is not ideal. But I don't know how to solve it. The way margins work in the PDF theming system is nothing like in HTML. In short, we don't have the same capabilities here as HTML/CSS does in that we don't have margin collapsing. There's a possibility that the converter could keep track of the last margin it applied, and roll it back if it falls at the end of the cell. But that's a big if. I'm not sure if the converter has he opportunity in the call chain to do that since the table is being rendered by a different library (prawn-table).

To be clear, it's not an empty blank line. It's a bottom margin that isn't being collapsed.

**Comment by mojavelinux** [comment]
This happens specifically for AsciiDoc table cells. It's also not related to the default theme. It affects all themes because it's how the margin is applied, not how the margin is defined.

**Comment by janicemhdc** [comment]
Well that's unfortunate.

**Comment by mojavelinux** [comment]
I went for a stroll and I'm happy to say I may have figured out how to make this work in a simple way. I'm still in the process of proving the idea. Refer to #1515 to follow my progress.

**Comment by janicemhdc** [comment]
Great! I've followed that issue. Too bad it's not backwards compatible, but such is life.

**Comment by mojavelinux** [comment]
Backwards compatibility is not yet out of reach ;) We'll have to see how it
turns out.

**Comment by CWempe** [comment]
I just want to leave an example, because I have the same issue.
It is unfortunate it cannot be solved before v2.0.0. 

Example:

```
|=== 

h| Header 1 h| Header 2 h|Header 3

| Row 1
| Row 1
| Row 1

| Row 2
a| Row 2
| Row 2

|=== 
```

![image](https://user-images.githubusercontent.com/1270359/88491530-9be02b80-cfa3-11ea-807a-6dc8a9619625.png)

**Comment by mojavelinux** [comment]
Thanks for the use case.

I'm not really working on 1.5.x anymore. All eyes are on 2.0.0. But I don't expect the transition to be difficult.

**Comment by mojavelinux** [comment]
This issue is being addressed by #1515.
